src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1590,7 +1590,7 @@ int wm_vuldet_linux_rm_false_positivies(sqlite3 *db, agent_software *agent, OSHa
                     wdb_finalize(stmt);
                 }
 
-                if (pkg->bin_name && !pkg->discard) {
+                if (!pkg->discard) {
```

Comment:
CID 210507

Coverity assumes `pkg->bin_name` can be NULL due to it is being checked here. So, it complains when it is passed to `strcmp()`. I have removed the check since we know it cannot be NULL.